.github/workflows/docker.yml

```diff
@@ -46,16 +46,14 @@ jobs:
         include:
           - name: Check
             file: "scripts/check-container.dockfile"
-            args: "cpu_target=avx"
             tags: "ghcr.io/facebookincubator/velox-dev:check-avx"
```

Comment:
Should we rename the tag to check from check-avx then ?

Reply:
Fixed